addons/stock/tests/test_move2.py

```diff
@@ -1124,17 +1125,18 @@ def test_backorder_4(self):
 
     def test_assign_deadline(self):
         """ Check if similar items with shorter deadline are prioritized. """
+        # Avoid to merge move3 and move4 for the test case
+        # TODO: should this be done as sudo?
```

Comment:
Ir parameter need to be created by admin. Since the demo user doesn't have admin right yes. For the test we can use sudo. If you want to simulate a real life case, you should create it with an admin user